about_blocks.rb

```diff
@@ -40,8 +40,8 @@ def many_yields
   def test_methods_can_call_yield_many_times
     result = []
     many_yields { |item| result << item }
-    assert_equal __, result
-  end
+    assert_equal [:peanut, :butter, :and, :jelly], result
```

Comment:
You should use %i[peanut, butter, and, jelly] instead of "[:peanut, :butter, :and, :jelly]" :D.